app/assets/javascripts/admin/spree/orders/use_billing.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
+  var order_use_billing_input = $('input#order_use_billing');
+
+  var order_use_billing = function () {
+    if (!order_use_billing_input.is(':checked')) {
+      $('#shipping').show();
+    } else {
+      $('#shipping').hide();
+    }
+  };
+
+  order_use_billing_input.click(function() {
+    order_use_billing();
+  });
+
+  order_use_billing();
+});
```

Comment:
Not a big deal, but there's a missing newline here

Reply:
this is fixed already in a later commit :+1: